integrations/axum/src/lib.rs

```diff
@@ -1079,7 +1124,11 @@ pub trait LeptosRoutes {
 /// The default implementation of `LeptosRoutes` which takes in a list of paths, and dispatches GET requests
 /// to those paths to Leptos's renderer.
 impl LeptosRoutes for axum::Router {
-    fn leptos_routes<IV>(
+      #[cfg_attr(
+        any(debug_assertions, feature = "ssr"),
```

Comment:
As with Actix, can remove the check for SSR here and anywhere else in the Axum integration.